src/Akka.Streams.Kafka.Tests/KafkaFixture.cs

```diff
@@ -72,6 +72,15 @@ await CreateContainer(KafkaImageName, KafkaImageTag, _kafkaContainerName, KafkaP
                 ["KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR"] = "1"
             });
 
+            /*
+            // Remove old networks, if there are any
+            var networks = await _client.Networks.ListNetworksAsync(new NetworksListParameters());
+            foreach (var existingNetwork in networks.Where(n => n.Name.StartsWith("network-")))
+            {
+                await _client.Networks.DeleteNetworkAsync(existingNetwork.ID);
+            }
+            */
```

Comment:
This part is mostly for local debugging.
When tests are aborted, no cleanup code is executed and some resources are still sitting in the system.
While removing containers is pretty simple using docker Kitemaric UI, networks cleanup should be performed manually. So, this code part helps.
Maybe in future I will create PR performing cleanup on tests setup before allocating new resources.

Reply:
Added issue #46 to keep track on this.